src/Blueprint/Blueprint.Test/ScenarioTests/BlueprintAssignmentTests.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.Commands.ScenarioTest;
+using Microsoft.WindowsAzure.Commands.ScenarioTest;
+using Xunit;
+
+namespace Microsoft.Azure.Commands.Blueprint.Test.ScenarioTests
+{
+    public class BlueprintAssignmentTests
+    {
+        private ServiceManagement.Common.Models.XunitTracingInterceptor _logger;
+
+        public BlueprintAssignmentTests(Xunit.Abstractions.ITestOutputHelper output)
+        {
+            _logger = new ServiceManagement.Common.Models.XunitTracingInterceptor(output);
+            ServiceManagement.Common.Models.XunitTracingInterceptor.AddToContext(_logger);
+            TestExecutionHelpers.SetUpSessionAndProfile();
+        }
+
+        [Fact]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
```

Comment:
Ooh... what are these?

Reply:
This is to mark the test to be run during CI in playback mode. [https://github.com/Azure/azure-powershell/blob/89eea9c118fa45f05f04e5dd09dc285f4a70a22e/documentation/development-docs/azure-powershell-developer-guide.md](https://github.com/Azure/azure-powershell/blob/89eea9c118fa45f05f04e5dd09dc285f4a70a22e/documentation/development-docs/azure-powershell-developer-guide.md)

---
In reply to: [257123201](https://github.com/Azure/azure-powershell/pull/8537#discussion_r257123201) [](ancestors = 257123201)